app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js

```diff
@@ -19,20 +19,21 @@ class ExpandedStoryAttachments extends React.Component {
     if (rejectedFiles.length > 0) return;
 
     const newFile = files[0];
-    const { onAdd } = this.props;
-
+    const { onAdd, onLoading } = this.props;
+    
     this.setState({ loading: true },
-      () => {
-        upload(newFile)
-          .then(attachment =>
-            onAdd(attachment)
+      () => { 
+          onLoading();
```

Comment:
Can we rename this prop to `onStartLoading`?

Reply:
sure thing!